app/celery/tasks.py

```diff
@@ -391,6 +393,9 @@ def update_letter_notifications_to_error(self, notification_references):
     )
 
     current_app.logger.info("Updated {} letter notifications to technical-failure".format(updated_count))
+    notifications = dao_get_notifications_by_reference(references=notification_references)
```

Comment:
We should probably check for the service_callback_api before creating this task.

Reply:
changed in 3594fc.